xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm

```diff
@@ -33,14 +33,24 @@
   <input type="text" id="xwikidocparentinput" name="parent" value="$!{escapetool.xml("$!docParent")}" size="30" class="suggestDocuments "/>
 </div>
 #if ($editor != 'inline')
-<div id="titleinput" class="form-group">
-  <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
-  <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
-</div>
+<dl id="titleinput" class="form-group">
+  <dt>
+    <label for="xwikidoctitleinput" value="$escapetool.xml("$!docTitle")">$services.localization.render('core.editors.content.titleField.label')</label>
```

Comment:
The ``value`` attribute on the ``label`` element doesn't make sense. Also, this line is too long and the label text is not XML escaped using ``$escapetool.xml()``.

Reply:
Sorry I feel very dumb for doing that.
 Can you check it now?